src/dynamics/joint/joint_set.rs

```diff
@@ -76,6 +76,12 @@ impl JointSet {
         self.joint_graph.graph.edge_weight(*id)
     }
 
+    /// Gets a mutable reference to the joint with the given handle.
+    pub fn get_mut(&mut self, handle: JointHandle) -> Option<&mut Joint> {
+        let id = self.joint_ids.get_mut(handle.0)?;
```

Comment:
This does not have to be mutable:

```suggestion
        let id = self.joint_ids.get(handle.0)?;
```

Reply:
ah no, of course